fault_injection/fi_gear/dummy/dummy_emfi.py

```diff
@@ -0,0 +1,76 @@
+                 x_position_step: int, y_position_min: int, y_position_max: int,
+                 y_position_step: int, voltage_min: int, voltage_max: int,
+                 voltage_step: int, pulse_width_min: int,
+                 pulse_width_max: int, pulse_width_step: int,
+                 trigger_delay_min: int, trigger_delay_max: int,
+                 trigger_step: int):
+        self.x_position_min = x_position_min
+        self.x_position_max = x_position_max
+        self.x_position_step = x_position_step
+        self.y_position_min = y_position_min
+        self.y_position_max = y_position_max
+        self.y_position_step = y_position_step
+        self.voltage_min = voltage_min
+        self.voltage_max = voltage_max
+        self.voltage_step = voltage_step
+        self.pulse_width_min = pulse_width_min
+        self.pulse_width_max = pulse_width_max
+        self.pulse_width_step = pulse_width_step
+        self.trigger_delay_min = trigger_delay_min
+        self.trigger_delay_max = trigger_delay_max
+        self.trigger_step = trigger_step
+
+    def arm_trigger(self, fault_parameters: dict) -> None:
+        """ Arm the trigger.
+
+        Args:
+            A dict containing the FI parameters.
+        """
+        print(f"Arming DummyEMFI trigger with x_position="
+              f"{fault_parameters['x_position']} y_position="
+              f"{fault_parameters['y_position']}, voltage="
+              f"{fault_parameters['voltage']}, pulse_width="
+              f"{fault_parameters['pulse_width']}, and trigger_delay="
+              f"{fault_parameters['trigger_delay']}")
+
+    def generate_fi_parameters(self) -> dict:
+        """ Generate EMFI parameters within the provided limits.
+
+        Returns:
+            A dict containing the FI parameters.
+        """
+        parameters = {}
+        parameters["x_position"] = random_float_range(self.x_position_min,
+                                                      self.x_position_max,
+                                                      self.x_position_step)
+        parameters["y_position"] = random_float_range(self.y_position_min,
+                                                      self.y_position_max,
+                                                      self.y_position_step)
+        parameters["voltage"] = random_float_range(self.voltage_min,
+                                                   self.voltage_max,
+                                                   self.voltage_step)
+        parameters["pulse_width"] = random_float_range(self.pulse_width_min,
+                                                       self.pulse_width_max,
+                                                       self.pulse_width_step)
+        parameters["trigger_delay"] = random_float_range(self.trigger_delay_min,
+                                                         self.trigger_delay_max,
+                                                         self.trigger_step)
```

Comment:
Do you also want to add an example for a deterministic XY sweep?

Reply:
I've added an example for a deterministic XY sweep to LFI & EMFI.